packages/angular/cli/lib/config/schema.json

```diff
@@ -1063,62 +1063,62 @@
             },
             "open": {
               "type": "boolean",
-              "description": "Opens the url in default browser.",
+              "description": "When true, open the live-reload URL in default browser.",
               "default": false,
               "alias": "o"
             },
             "liveReload": {
               "type": "boolean",
-              "description": "Whether to reload the page on change, using live-reload.",
+              "description": "When true, reload the page on change using live-reload.",
               "default": true
             },
             "publicHost": {
               "type": "string",
-              "description": "Specify the URL that the browser client will use."
+              "description": "The URL that the browser client will use for live reload."
```

Comment:
`publicHost` is not just about live-reload. @clydin can you offer a better description please?

Reply:
`for live reload` -> `to connect to the development server`

Reply:
The issue was that the user didn't relate publicHost to live-reload. How about  this:
"The URL that the browser client will use to connect to the development server for live-reload."

Reply:
@clydin Please check new text